man/systemd.network.xml

```diff
@@ -952,18 +952,21 @@ Table=1234</programlisting></para>
         <term><varname>ConfigureWithoutCarrier=</varname></term>
         <listitem>
           <para>Takes a boolean. Allows networkd to configure a specific link even if it has no carrier.
-          Defaults to false. If <option>IgnoreCarrierLoss=</option> is not explicitly set, it will
-          default to this value.
+          Defaults to false. If enabled, and the <varname>IgnoreCarrierLoss=</varname> setting is not
+          explicitly set, then it is enabled.
```

Comment:
maybe "then it is enabled as well" to indicate that it's talking about the second setting